packages/stacking/src/index.ts

```diff
@@ -504,6 +505,23 @@ export class StackingClient {
     );
   }
 
+  /**
+   * Get number of seconds until the end of the stacking deadline.
+   * This is the estimated time stackers have to submit their stacking
+   * transactions to be included in the upcoming reward cycle.
```

Comment:
The function returns a negative value during the prepare phase (after the stacking deadline and before users can stack again for the next cycle).